tools/kontaktdaten.py

```diff
@@ -383,6 +392,30 @@ def validate_wochentag(wochentag: str):
         raise ValidationError(str(exc)) from exc
 
 
+def validate_pushover_app_token(pushover_app_token: str):
+    if not isinstance(pushover_app_token, str):
+        raise ValidationError("Muss eine Zeichenkette sein")
+    if len(pushover_app_token) == 29:
```

Comment:
Der Check müsste so sein, wenn es genau 30 Zeichen lang sein soll, oder?

```suggestion
    if len(pushover_app_token) != 30:
```